log/logClient.go

```diff
@@ -406,7 +407,20 @@ func WatchTelemetryHelper(arr []byte, t string, o Options) {
 		o.EventChan <- EventInfo{Data: arr, Type: t}
 	}
 
-	if o.JSON {
+	if o.JSON || o.Output == "json" {
+		str = fmt.Sprintf("%s\n", string(arr))
+	} else if o.Output == "pretty-json" {
+
+		var prettyJSON bytes.Buffer
+		err = json.Indent(&prettyJSON, arr, "", "  ")
+
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Failed to prettify JSON (%s)\n", err.Error())
+		}
+		str = fmt.Sprintf("%s\n", prettyJSON.String())
+
+	} else if o.Output == "json" {
+
```

Comment:
This is redundant now, since we added it in the earlier check

Reply:
Thanks for pointing out, forgot that!